ext/SymbolicRegressionPlotsExt.jl

```diff
@@ -0,0 +1,43 @@
+module SymbolicRegressionPlotsExt
+
+import Plots: plot
+import SymbolicRegression: default_sr_plot
+
+using DynamicExpressions: Node
+using SymbolicRegression: HallOfFame, Options, string_tree
+using SymbolicRegression.HallOfFameModule: format_hall_of_fame
+
+function plot(hall_of_fame::HallOfFame, options::Options; variable_names=nothing, kws...)
+    return default_sr_plot(hall_of_fame, options; variable_names, kws...)
+end
+
+function default_sr_plot(
+    hall_of_fame::HallOfFame, options::Options; variable_names=nothing, kws...
+)
+    (; trees, losses, complexities) = format_hall_of_fame(hall_of_fame, options)
+    return default_sr_plot(trees, losses, complexities, options; variable_names, kws...)
+end
+
+function default_sr_plot(
+    trees::Vector{N},
+    losses::Vector{L},
+    complexities::Vector{Int},
+    options::Options;
+    variable_names=nothing,
+    kws...,
+) where {T,L,N<:Node{T}}
+    tree_strings = [string_tree(tree, options; variable_names) for tree in trees]
+    return plot(
```

Comment:
@eelregit this is the current plotting code which uses Plots.jl. Do you want to figure out how to add your convex hull to this?

Reply:
Looks like there’s a Julia Convex Hull implementation here: https://juliareach.github.io/LazySets.jl/v1.11/man/convex_hulls.html#D-convex-hull-1